release_notes/ocp_3_10_release_notes.adoc

```diff
@@ -54,33 +54,524 @@ documentation.
+
+* Some nodes are under- or over-utilized.
+* The original scheduling decision does not hold true any more, as taints or
+labels are added to or removed from nodes, pod/node affinity requirements are
+not satisfied any more.
+* Some nodes failed and their pods moved to other nodes.
+* New nodes are added to clusters.
+
+[[ocp-310-node-problem-detector]]
+==== Node Problem Detector (Technology Preview)
+The Node Problem Detector is in xref:ocp-310-technology-preview[Technology
+Preview] and is not for production workloads. The Node Problem Detector monitors
+the health of your nodes by finding certain problems and reporting these
+problems to the API server, where external controllers could take action.
+
+[[ocp-310-system-services-now-hosted-on-pods]]
+==== System Services Now Hosted on Pods
+Each of the system services, API, controllers, and etcd, used to run as system
+services on the master. These services now run on static pods in the cluster. As
+a result, there are new commands to restart these services: `master-restart
+api`, `master-restart controllers`, and m`aster-restart etcd`. To view log
+information on these services, use `master-logs api api`, `master-logs
+controllers controllers`, and `master-logs etcd etcd`.
+
+[[ocp-310-new-node-configuration-process]]
+==== New Node Configuration Process
+You can modify exiting nodes through a configuration map rather than the
+*_node-config.yaml_*. The installation creates three node configuration groups:
+*node-config-master*, *node-config-infra*, and *node-config-compute* and creates
+a configuration map for each group.  A sync pod watches for changes to these
+configuration maps. When a change is detected, the sync pod updates the
+*_node-config.yaml_* file on all of the nodes.
+
+[[ocp-310-group-pruning]]
+==== Group Pruning
+To prune groups records from an external provider, administrators can
+run the following command:
+
+----
+$ oc adm prune groups --sync-config=path/to/sync/config [<options>]
+----
+See xref:../admin_guide/pruning_resources.adoc#pruning-groups[Pruning groups] for more information.
+
 [[ocp-310-installation]]
 === Installation
 
+[[ocp-310-important-installation-changes]]
+==== Important Changes
+
+A number of important changes have been made to the installation process in
+{product-title} 3.10.
+
+* The control plane components (etcd, API server, and controllers) are now run as
+static pods by the kubelet on the masters. This is now the only supported way
+they can be run. The upgrade will change over to this automatically. You can see
+the pods once the control plane comes up with `oc get pods -n kube-system` and
+execute, log, and inspect them using normal `oc` CLI commands.
+
+* The containerized mode for {product-title} is no longer supported (where
+ components run in docker containers) and the 3.10 upgrade will require you to
+ move to RPMs or system containers for the kubelet by setting
+ `openshift_use_system_containers=true`.
```

Comment:
`openshift_use_system_containers` is removed in https://github.com/openshift/openshift-ansible/pull/8329.

The changes will be done automatically according to the OS during the upgrade:

- Atomic Host: Node service run as system container
- RHEL: Node service run as rpm